example/leaflet-zoomdisplay.html

```diff
@@ -3,14 +3,14 @@
 <head>
     <meta charset="UTF-8">
     <title>Leaflet.zoomdisplay Example</title>
-    <link rel="stylesheet" href="../lib/leaflet/leaflet.css" />
+    <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" />
     <!--[if lte IE 8]><link rel="stylesheet" href="../libs/leaflet/leaflet.ie.css" /><![endif]-->
```

Comment:
Lets remove this IE specific line, since they no longer have a separate IE stylesheet

https://cdnjs.com/libraries/leaflet/